aiohue/sensors.py

```diff
@@ -53,3 +71,367 @@ def state(self):
     @property
     def config(self):
         return self.raw['config']
+
+
+class GenericCLIPSensor(GenericSensor):
+    def __init__(self, id, raw, request):
+        super().__init__(id, raw, request)
+        self._on = raw['config'].get('on')
```

Comment:
Since we store raw as instance variable, we should have properties just fetch it from `raw`. That way we keep a single source of truth.